linux/open_vm_tools/install_ovt.yml

```diff
@@ -52,3 +52,18 @@
 # Eject CDROM /dev/sr0 from guest
 - include_tasks: ../utils/eject_cdrom_in_guest.yml
   when: guest_os_ansible_distribution in ['SLES', 'SLED', 'RedHat']
+
+# Reboot and Wait for tools running
+- include_tasks: ../utils/vm_reboot.yml
+
+# Get VMware Tools version and build
+- include_tasks: ../utils/get_guest_ovt_version_build.yml
+
+# Get the OS built-in open-vm-tools version
+- name: "Set guest inbox open-vm-tools version"
+  set_fact:
+    guest_inbox_ovt_version: "{{ vmtools_info_from_vmtoolsd }}"
```

Comment:
Here is there a condition that if using online official repo, the VMware tools is updated, not the built-in OVT?